server-spi-private/src/main/java/org/keycloak/validate/ValidationResult.java

```diff
@@ -31,6 +31,11 @@
      */
     public static final ValidationResult OK = new ValidationResult(true, Collections.emptySet());
 
+    /**
+     * An empty ValidationResult that's invalid by default.
+     */
+    public static final ValidationResult INVALID = new ValidationResult(false, Collections.emptySet());
```

Comment:
Why should it be allowed to have an invalid result but no validation errors? Would be less surprising that an invalid result is always based on validation errors.